scripts/jenkins/cloud/ansible/roles/crowbar_reapply_barclamps/defaults/main.yml

```diff
@@ -0,0 +1,19 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+crowbar_proposal_name:
+  pacemaker: "services"
```

Comment:
You need to get the actual list of pacemaker proposals. This only works if you have one cluster, but remember that we also support std-split and other scenarios that split the control plane into several clusters.

Reply:
Is this just for pacemaker or other barclamps can also have other proposals besides the `default` ?

Reply:
no, pacemaker is "special" that way

Reply:
alright, will make that change. thanks!

Reply:
Turns out that reapplying the barclamp proposals are not the right procedure to update. This commit was dropped from that PR and will be made in a following PR.